content/projects/gsoc/2025/project-ideas.html.haml

```diff
@@ -26,6 +26,15 @@ opengraph:
     width: 175px;
   }
 
+  @media screen and (max-width: 450px) {
+    .pi_table {
+      display: block;
+      width: 100%;
+      overflow-x: auto;
+      -webkit-overflow-scrolling: touch;
```

Comment:
`-webkit-overflow-scrolling: touch;` is non-standard and [dead](https://developer.apple.com/documentation/safari-release-notes/safari-13-release-notes#New-Features)